### ISSUE TITLE
if enabled this plugin seemingly forces "file was modified externally, merging" notifications and issues along that

### ISSUE THREAD

**Issue by bietiekay** [mnaoumov/obsidian-backlink-cache#4]
After some recent updates this plugin is the reason for my fairly sizeable vault (10k notes, 100k links) to produce errors while I type normal text.

The error is like this: 

I type a sentence and while typing Obsidian will show a notification "the file has been changed externally and was automatically merged". Depending on how fast I type the cursor will be positioned further left in the line and I type "over" the text I just had typed due to this merging-delay(?).

When I disable the backlink cache plugin, this notification will not show up, no matter how fast I type.

Thus --> this plugin creates a condition where modifications are being detected and merged by obsidian causing bad typing/writing experience further.

**Comment by mnaoumov** [comment]
@bietiekay please check if the issue is still happening with the latest version of the plugin

**Comment by ksdavidc** [comment]
Yes, this is still happening. If this notification is being created by your plugin, it would be helpful if you would include that in the message, as it took me a while to determine WHICH plugin was sending the notification.

I  do have a hint for you: it seems to be related to having the same file opened in two tabs, at least for me. Haven't tested this thoroughly, though.

**Comment by mnaoumov** [comment]
@ksdavidc no, this message is created by Obsidian itself. The plugin doesn't change notes, so it's very unlikely it is the cause of the problem

**Comment by ksdavidc** [comment]
Possibly. However, both the previous poster and I both found that disabling your plugin stops the messaging from appearing. So even if the bug doesn't originate in your plugin, it seems that ONLY your plugin is bringing the message to the fore. Disabling it, even with over a hundred other plugins does not produce the error.

attached is sandbox vault with only backlink cache plugin installed.


[backlink cache.zip](https://github.com/user-attachments/files/20430143/backlink.cache.zip)


Also, video:

https://github.com/user-attachments/assets/1894c6c3-5e27-487e-ba59-cb86e349ff9c

I appears I had to open the second tab again, i.e. it doesn't trigger from startup, possibly

(just to be perfectly clear: using just a vanilla sandbox vault and following the same steps and typing at length at full speed, I was unable to evince the error. )

**Comment by mnaoumov** [comment]
@ksdavidc thanks, I could identify the issue

**Comment by mnaoumov** [comment]
Fixed in https://github.com/mnaoumov/obsidian-backlink-cache/releases/tag/2.8.2

**Comment by ksdavidc** [comment]
thx
